tests/contrib/django/django1_app/settings.py

```diff
@@ -45,6 +47,9 @@
     },
 }
 
+if django.VERSION >= (4, 0, 0):
+    CACHES["redis"]["BACKEND"] = "django.core.cache.backends.redis.RedisCache"
```

Comment:
it might be cleaner to organize as... before `CACHES` is defined:

```python
redis_backend = "django_redis.cache.RedisCache"
if django.VERSION >= (4, 0, 0):
    redis_backend = "django.core.cache.backends.redis.RedisCache"

CACHES = {
    ...
    "redis": {
        "BACKEND": redis_backend,
        "LOCATION": "..."
    },
    ...
}
```

a little cleaner than reaching into the nested dict later